rust/src/enip/enip.rs

```diff
@@ -318,6 +315,7 @@ impl EnipState {
                             tx.response = Some(pdu);
                         }
                         self.transactions.push_back(tx);
+                        sc_app_layer_parser_trigger_raw_stream_reassembly(flow, dir as i32);
```

Comment:
Should it be done for each tx ? Or outside the loop for a bunch of tx ?

Reply:
Thank you. I should revisit this. It should ideally be done on completion of each request or response.